### PR TITLE
Update for PureScript 0.7.3 and add evalM

### PR DESCRIPTION
**Pull request by garyb**

### CODE REVIEW

**Comment by garyb** [comment]
@cryogenian I removed the `inPhantom` function from the tests as it was hiding errors that occurred during running, and it didn't really seem like it was doing anything that we need during the normal usage of the test suite - I assume it was something for `phantomjs`, but we test in `node` here anyway?

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ node_js:
 install:
   - npm install gulp bower -g
   - npm install
-  - bower update
+  - bower install --production
 script:
   - gulp
+  - bower install
+  - gulp test
```
Comment (garyb) [inline .travis.yml]: I've started updating travis files to do this when we have `devDependencies` as it ensures we're not accidentally depending on a `devDependency` in the normal `make` too then.

**Comment by cryogenian** [comment]
@garyb True, there is no need of `phantomjs` (I used to run tests on it, but don't remeber why)

**Comment by garyb** [comment]
Ok, cool. Just wanted to check!

**Pull request closed without merging**